pyuvdata/uvdata.py

```diff
@@ -71,7 +71,18 @@ def __init__(self):
                                           acceptable_vals=["uncalib", "Jy", "K str"])
 
         desc = ('Number of data points averaged into each data element, '
-                'NOT required to be an integer. type = float, same shape as data_array')
+                'NOT required to be an integer, type = float, same shape as data_array.'
+                'The product of the integration_time and the nsample_array '
+                'value for a visibility reflects the total amount of time '
+                'that went into the visibility. Best practice is for the '
+                'nsample_array to be used track flagging within an integration_time '
```

Comment:
Small typo: "to be used _to_ track flagging"

Reply:
Thanks!